services/movie_session.py

```diff
@@ -42,3 +42,14 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict[str, int]]:
```

Comment:
The type hint `list[dict[str, int]]` is only supported in Python 3.9 and later. If you need compatibility with earlier Python versions, consider using `List[Dict[str, int]]` from the `typing` module.